raiden-ts/src/helpers.ts

```diff
@@ -357,3 +357,18 @@ export async function waitConfirmation(
     )
     .toPromise();
 }
+
+/*
+ * Returns true if `url` is a valid URL or domain.
+ * On production `https://` is required for URLs, otherwise `http://` matches as well.
+ *
+ * @param url - A URL or hostname
+ * @returns true if valid URL or domain
+ */
+export const isValidUrl = (url: string): boolean => {
+  const regex =
+    process.env.NODE_ENV === 'production'
+      ? /^(?:https:\/\/)?[^\s\/$.?#&"']+\.[^\s\/$?#&"']+$/
+      : /^(?:(https|http):\/\/)?[^\s\/$.?#&"']+\.[^\s\/$?#&"']+$/;
```

Comment:
You have to allow the port number too in the `development` environment the container running the tests has pfs at `http://localhost:6000` this should also be considered a valid URL in develop

Reply:
I guess makes sense for production as well?

Reply:
@nephix no not really. In production this would be https where the port `443` is omitted